<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSBizSettingBasic</class>
 <widget class="QWidget" name="GBSBizSettingBasic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="frmConsole" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1139</width>
     <height>599</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 4px;
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>32</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>基础</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>推流直播</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>19</y>
      <width>2</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxStartLive">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>142</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>开始直播时弹窗确认</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>142</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>停止直播时弹窗确认</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxStopLive">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>142</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>停止录制时弹窗确认</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxStopRecord">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxAutoRecord">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>142</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>直播时自动录制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>135</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>推流码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>88</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>推流码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>221</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>221</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>50</y>
      <width>36</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>拉流地址</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>135</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>拉流地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>611</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
    <property name="text">
     <string>1212</string>
    </property>
    <property name="placeholderText">
     <string>  应用于手机虚拟摄像头或者其他直播设备拉流</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>身份验证</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>351</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
    <property name="placeholderText">
     <string>  账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>260</y>
      <width>341</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
    <property name="placeholderText">
     <string>  密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>260</y>
      <width>36</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
